include/ccf/ds/logger.h

```diff
@@ -387,6 +387,8 @@ namespace logger
     CCF_LOGGER_DEPRECATE(LOG_TRACE_FMT) CCF_LOG_FMT(TRACE, "")
 #  define LOG_DEBUG_FMT \
     CCF_LOGGER_DEPRECATE(LOG_DEBUG_FMT) CCF_LOG_FMT(DEBUG, "")
+#  define LOG_DEV_FMT \
+    CCF_LOGGER_DEPRECATE(LOG_TRACE_FMT) CCF_LOG_FMT(TRACE, "")
```

Comment:
Might as well do this at INFO level, so you don't _also_ need verbose logging enabled? And making the second argument `"dev"` will add a `[dev]` tag making them more easily greppable.